lib/rubocop/cop/layout/multiline_method_call_indentation.rb

```diff
@@ -182,7 +182,7 @@ def semantic_alignment_base(node, rhs)
           return unless rhs.source.start_with?('.', '&.')
 
           node = semantic_alignment_node(node)
-          return unless node&.loc&.selector
+          return unless node&.loc&.selector && node.loc.dot
```

Comment:
memo) It would be better to merge this PR after merging https://github.com/rubocop/rubocop/pull/12266 , since it needs the same modification as https://github.com/rubocop/rubocop/pull/12266 .